setup/kubernetes/local-preview.md

```diff
@@ -52,15 +52,16 @@ apply):
 
 1. Docker Desktop for Mac **must** use version [2.5.0.1][docker-mac-url]. This
    specific version is required because of a recent downgrade to Linux Kernel
-   4.9 due to a [bug](docker-bug-url).
+   4.9 due to a [bug][docker-bug-url].
 
 ### Dev URLs
 
-Currently, the local preview doesn't support [dev URLs][devurl-url]. Instead,
-you can use tools like [ngrok][ngrok-url] to preview webpages from inside an
-workspace.
+The local preview doesn't support [dev URLs](../workspaces/devurls) by default.
+Instead, you can use tools like [ngrok][ngrok-url] to preview webpages from
+inside a workspace.
 
-We are working on bringing Dev URL support to local previews in later releases.
+To enable local dev URLs, follow
+[the instructions below](#enable-local-dev-urls) after installing Coder.
```

Comment:
This seems a bit weird to say devurls aren't supported, then giving instructions on how to enable them.

Reply:
agreed, just reworded